documentation/asciidoc/topics/proc_disabling_gossip_router_cross_site.adoc

```diff
@@ -0,0 +1,46 @@
+[id='disabling-gossip-router-cross-site_{context}']
+= Disabling local Gossip router and service
+
+[role="_abstract"]
+The {ispn_operator} starts a Gossip router on each site, but you only need a single Gossip router to manage traffic between the {brandname} cluster members.
+You can disable the additional Gossip routers to save resources.
+
+.Prerequisites
+
+* Have multiple working Gossip routers.
```

Comment:
I don't multiple Gossip Routers is a prerequisite. Rather, it's necessary for a Gossip Router to be enabled on at least one site.